src/test/java/com/swiftype/appsearch/ClientTest.java

```diff
@@ -80,6 +80,20 @@ void listEngines() throws ClientException {
       .collect(Collectors.toList());
     assertEquals(enginesWithEngineName.size(), 1);
   }
+  
+  @Test
+  void listEnginesWithPagination() throws ClientException {
+    List<Map<String, Object>> engines = client.listEngines(1, 20);
+    client.createEngine(engineName);
+    List<Map<String, Object>> enginesAfterCreate = client.listEngines(1, 20);
+
+    assertEquals(enginesAfterCreate.size() - 1, engines.size());
```

Comment:
this isn't actually the expected response - the expected response includes `meta` and `results` as top level keys